Auto delete

nithinvadakken opened the pull request.

MeiH10 reviewed (changes requested):
I thought we moved deleteEventByUser() and addEventToDatabase() to separate files in useful script. Also I thought we call startSync in sqldb.js  not index.js

nithinvadakken commented:
> I thought we moved deleteEventByUser() and addEventToDatabase() to separate files in useful script. Also I thought we call startSync in sqldb.js not index.js

removed from index.js, got caught in when merging

nithinvadakken commented:
is there anything else needed for this to be merged?
